### DISCUSSION TITLE
Get on Twitch

### DISCUSSION THREAD

**Discussion by TheJaredWilcurt** [EddieHubCommunity/support#213] [Ideas] [answered]
* When someone follows you on Twitch they automatically get mobile notifications when you go live if they have the Twitch app. With YouTube live streams notifications never pop up for me.
* Twitch also has considerably better mod and bot tooling options.
* It is common at the end of a Twitch stream to "raid" similar streams, passing your viewers off to someone else. This helps build community and when you are raided it introduces new viewers to your content.
* There are better tools for controlling chat, such as slow-mo mode.
* Nightbot is built in and allows for common commands like `!youtube` or `!discord` for the bot to reply with links for those new in chat to join/follow.
* The emotes on YouTube are terrible. There is no personality and the fact that the most commonly used emote on the platform is the "Loading spinner" says a lot about the quality of streams.
* There are multi-broadcast tools, but they are more effort to set up and give a worse experience to the audience as the streamer neglects one platforms chat for the other, and people in chat can't communicate with each other.
* You can try splitting different content up between the platforms. Example: Faraday academy does videos and live interviews on YouTube, and live coding streams on Twitch.

**Comment by eddiejaoude** [comment, marked as answer]
Thank you @TheJaredWilcurt for the feedback and tips 👍 - also for being in our live stream today with @FaradayAcademy.

I used to stream on Twitch before YouTube. I think II could stream different content like you said between the platforms. Definitely something I should look into 🤓
